railties/lib/rails/application/configuration.rb

```diff
@@ -489,6 +490,11 @@ def default_log_file
         f
       end
 
+      def default_log_size
+        # Defaults to 100 MB
+        (100 * 1024 * 1024) if Rails.env.development? || Rails.env.test?
```

Comment:
I don't like that we're checking `Rails.env.development?` from here. Can we instead add this in the `development.rb` and `test.rb` templates?

Or maybe do this from `Configuration#load_defaults`?

Reply:
sounds good, I was looking for something like that